src/oops/designPatterns/prototype/Registry.java

```diff
@@ -11,9 +11,11 @@
 //Singleton
 public class Registry {
 
-	private Map<String, Item> items = new HashMap<String, Item>();
+	private static Map<String, Item> items = new HashMap<String, Item>();
 	
-	public Registry() {
+	private static Registry registeryInstance = new Registry();
```

Comment:
We should not create an object when class loads. In this class when class loads it creates an instance of Registry and assigns it to registryInstance. It would have been null.